tests/slow/src/test/scala/tests/feature/DefinitionCrossLspSuite.scala

```diff
@@ -77,7 +77,7 @@ class DefinitionCrossLspSuite
       _ <- server.didOpen("scala/Predef.scala")
       _ = assertNoDiff(
         client.workspaceMessageRequests,
-        ""
+        "Preparing presentation compiler"
```

Comment:
Unless I'm mistaken, this should have been happening all the time, but may not have been due to some misconfiguration. Mainly because looking at this, if this runs without the current compiler version, then it uses the embedded presentation compile, which should be happening here, and that's where we see the "Preparing presentation compiler". This was failing without this, and that makes sense right?

Reply:
Totally. Let me look a bit closer at this.